regression-test/suites/nereids_syntax_p0/test_regexp_replace.groovy

```diff
@@ -36,4 +36,24 @@ suite("test_regexp_replace") {
         (5, null);"""
 
     qt_replace_in_table_chinese """SELECT id, regexp_replace(name, '\\\\p{Han}', '汉') as replaced_name FROM test_table_for_regexp;"""
+
+    qt_replace_ignore1 """select regexp_replace('{"abc":5},{"def":78}', '\\}\\,\\{', '\\}&&\\{', 'IGNORE_INVALID_ESCAPE');"""
+    qt_replace_ignore2 """select regexp_replace('abc', 'b', "\\}", 'IGNORE_INVALID_ESCAPE');"""
+    qt_replace_ignore3 """select regexp_replace_one('{"abc":5},{"def":78}', '\\}\\,\\{', '\\}&&\\{', 'IGNORE_INVALID_ESCAPE');"""
+    qt_replace_ignore4 """select regexp_replace_one('abc', 'b', '\\}', 'IGNORE_INVALID_ESCAPE');"""
```

Comment:
how this sql could exec since u not register regexp_replace_one in `fe/fe-core/src/main/java/org/apache/doris/catalog/BuiltinScalarFunctions.java`

Reply:
it's have register 
`scalar(RegexpReplaceOne.class, "regexp_replace_one"),`